compiler/src/main/java/io/neow3j/compiler/converters/ConstantsConverter.java

```diff
@@ -37,6 +40,15 @@ public AbstractInsnNode convert(AbstractInsnNode insn, NeoMethod neoMethod,
             case LDC:
             case LDC_W:
             case LDC2_W:
+                if (isDesiredAssertionStatusConst(insn)) {
+                    insn = insn.getNext();
+                    // Ignore instructions until static variable $assertionDisabled is loaded
+                    while (!isAssertionDisabledStaticField(insn)) {
+                        insn = insn.getNext();
+                    }
+                    insn.getNext();
```

Comment:
Shouldn't this be:
`insn = insn.getNext();`

Reply:
Yes, true, I missed that one.